src/components/Channel/ChatForm.jsx

```diff
@@ -0,0 +1,83 @@
+import { useState, useEffect } from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+
+import { socketClient } from '../../hooks/socket/useSocket';
+
+export default function ChatForm({ channelId }) {
+  const [input, setInput] = useState('');
+  const [userName, setUserName] = useState('');
+
+  useEffect(() => {
+    const { name } = JSON.parse(sessionStorage.getItem('user'));
+    setUserName(name);
+  }, []);
+
+  function submitChat(ev) {
+    ev.preventDefault();
+
+    const trimmedInput = input.trim();
+
+    if (!trimmedInput) return;
+
+    const newChat = {
+      author: userName.replace(' ', ''),
+      chat: trimmedInput,
+    };
+
+    socketClient.emit('new chat', { channelId, newChat });
+    setInput('');
+  }
+
+  function handleInputChange({ target }) {
+    const { value } = target;
+    setInput(value);
```

Comment:
```suggestion
    setInput(e.target.value);
```
이렇게 쓰는 것이 좀 더 이해하기 쉬울 것 같아요.